src/Components/Aspire.Microsoft.Azure.Cosmos/README.md

```diff
@@ -99,7 +99,7 @@ The .NET Aspire Microsoft Azure Cosmos DB library supports [Microsoft.Extensions
 You can also pass the `Action<AzureCosmosDBSettings> configureSettings` delegate to set up some or all the options inline, for example to disable tracing from code:
 
 ```csharp
-builder.AddAzureCosmosDB("cosmosConnectionName", settings => settings.Tracing = false);
+builder.AddAzureCosmosDbClient("cosmosConnectionName", settings => settings.Tracing = false);
```

Comment:
I don't think the regular DB components should have the name "Client" in them. These are the best names IMO:

* AddNpgsqlDataSource
* AddMySqlDataSource

and EF:

* AddSqlServerDbContext
* AddNpgsqlDbContext

Cosmos's class is named `CosmosClient`, so that's OK for it to be named Client.

The only one that is a bit off is `AddSqlServerClient`, but that's been that way for a while. It got named that way because the component is named `Aspire.Microsoft.Data.SqlClient`. And I don't think SqlClient supports a `SqlDataSource` yet. So it doesn't make sense for it to be called `AddSqlDataSource`.

So I think what we have in the code now is fine.

Reply:
My comment was about the casing of "DB" vs "Db" in the Cosmos client case. Or to remove the "DB" suffix altogether.

Reply:
`"DB"` is both the marketing case and the recommended .NET naming casing.

https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions#capitalization-rules-for-identifiers